sklearn/decomposition/tests/test_sparse_pca.py

```diff
@@ -70,6 +71,10 @@ def test_fit_transform():
     spca_lasso.fit(Y)
     assert_array_almost_equal(spca_lasso.components_, spca_lars.components_)
 
+    # Test that deprecated ridge_alpha parameter throws warning
+    assert_warns(DeprecationWarning, spca_lars.transform, Y, ridge_alpha=0.01)
+    assert_warns(DeprecationWarning, spca_lars.transform, Y, ridge_alpha=None)
```

Comment:
Just a niptick, it will be better to use `assert_warns_message` with a part of the message (just to be sure it's the right message).